data_diff/diff_tables.py

```diff
@@ -82,10 +84,62 @@ def _run_in_background(self, *funcs):
 class DiffResultWrapper:
     diff: iter  # DiffResult
     info_tree: InfoTree
+    stats: dict
 
     def __iter__(self):
         return iter(self.diff)
 
+    def print_stats(self, json_output: bool):
+
+        diff_by_key = {}
+        for sign, values in self.info_tree.info.diff:
```

Comment:
@erezsh I think I'm close but when running from CLI this `self.info_tree.info.diff` is None. I think it could be a race condition?

Reply:
You should be using `self.diff`. Info-tree only collects diff in the leaf nodes, not the branches/roots

Reply:
got it ty